2 - ListsLogicAndIterations/Code/Lists.cs

```diff
@@ -7,27 +7,39 @@ public static class Lists
 	{
 		public static List<decimal> BuildList(decimal first, decimal second)
 		{
-			throw new NotImplementedException();
+            var result = new List<decimal> { first, second };
+            return result;
 		}
 
 		public static decimal GetValue(List<decimal> list, int position)
 		{
-			throw new NotImplementedException();
+            try
+            {
+                return list[position];
+            }
+            catch
+            {
+                return 0;
+            }
```

Comment:
Once again, though this works relying on exception handling is expensive and is problematic. You should see if the operation is possible, vs trying and seeing if it worked. Example:

```
			if (position > (list.Count - 1) || position < 0)
			{
				return 0;
			}

			return list[position];
```